controllers/movieController.js

```diff
@@ -0,0 +1,22 @@
+const movieService = require('../services/movieService');
+
+const getMovies = async (req, res) => {
+  try {
+    const { main, release, sort } = req.body;
+
+    const filter = {
+      main: main,
+      release: release,
+      sort: sort,
+    };
+
+    let getMovies = await movieService.getMovies(filter);
+    return res.status(201).json(getMovies);
```

Comment:
```suggestion
    return res.status(200).json(getMovies);
```
GET 메소드와 더불어 현재 API의 기능 및 동작과 적합한 상태코드를 사용해주세요.